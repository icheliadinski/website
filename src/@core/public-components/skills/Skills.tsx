import React from 'react'
import { SideCard } from '@core/ui-kit'

export const Skills = () => {
  return (
    <SideCard>
      <article>My Skills</article>
    </SideCard>
  )
}
