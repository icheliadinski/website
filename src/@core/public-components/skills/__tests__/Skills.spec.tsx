import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Skills } from '../Skills'

describe('[Skills]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Skills />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
