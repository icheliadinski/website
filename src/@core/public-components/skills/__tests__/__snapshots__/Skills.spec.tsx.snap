// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Skills] renders successfully 1`] = `
<SideCard>
  <article>
    My Skills
  </article>
</SideCard>
`;
